### ISSUE TITLE
Custom Paginator

### ISSUE THREAD

**Issue by ajanini** [rebing/graphql-laravel#479]
### Versions:
- graphql-laravel Version: 1.23.0
- Laravel Version: 5.7.28
- PHP Version: 7.2.1

### Question:

I've followed the [example in the documentation](https://github.com/rebing/graphql-laravel/blob/v1/docs/advanced.md#customising-the-pagination-results) to implement a custom paginator class. My goal is to implement SEO-related metadata fields to my queries.

#### SeoPaginationFields Class
```php
namespace App\GraphQL\Support;

use GraphQL\Type\Definition\Type;

class SeoPaginationFields
{
    public static function getPaginationFields()
    {
        return [
            'seo_title' => [
                'type' => \Type::string(),
                'description' => 'SEO title',
                'resolve' => function() {
                    return 'foobar';
                },
                'selectable' => false,
            ],
        ];
    }

}
```

#### (relevant part of) graphql.php

```php
return [
    // ...
    'custom_paginators' => [
        'seo_pagination' => App\GraphQL\Support\SeoPaginationFields::class,
    ],
    // ...
];
```

#### Example GraphQL query
```graphql
query FetchRealties {
  realties (limit: 50) {
    data {
      id
      name
      permalink
    }
    total
    per_page
    seo_title
  }
}
```

#### Response
```graphql
{
  "errors": [
    {
      "message": "Cannot query field \"seo_title\" on type \"realtyPagination\".",
      "locations": [
        {
          "line": 10,
          "column": 5
        }
      ]
    }
  ]
}
```

---

I haven't *told* my query class to use this new pagination. Do I need to do that? How?

Also, if I add a `dd('foobar');` first thing in my `getPaginationFields` method, it is never triggered.

What am I missing?

Thanks!

**Comment by mfn** [comment]
This feature was removed some time ago (see https://github.com/rebing/graphql-laravel/commit/a9212adcda75b870965064c0113bd98b3b55203d ), no idea why.

But that explains why it's not working, the config does nothing anymore.

Also: your version is quite outdated, please upgrade to the latest stable.

**Comment by ajanini** [comment]
Thank you @mfn.

Any tips on how I could add SEO metadata to the the response from within the `resolve()` method of my queries? I've tried implementing a custom pagination `type` (as suggested [here](https://github.com/rebing/graphql-laravel/issues/159#issuecomment-467994693)), but I have yet to find a way to change my custom field's contents based on the current query.

Also, I've just upgraded to ver. 2.1.1 as you suggested.
